scRNA-seq/03-normalizing_scRNA.Rmd

```diff
@@ -159,20 +137,18 @@ The group normalization is then applied back to the individual cells within the
 In this case, we will also log-transform the normalized counts to get a less skewed distribution of expression measures. 
 Note that because of the zero counts, the `logNormCounts()` function will add a pseudocount of 1 to each value before computing the log.
 
-*These steps may take a minute or so.*
-
 ```{r sce_normalize}
 # Step 1) Group cells with other like cells by clustering.  
-qclust <- scran::quickCluster(gbm_sce)
+qclust <- scran::quickCluster(bladder_sce)
 
 # Step 2) Compute sum factors for each cell cluster grouping.  
-gbm_sce <- scran::computeSumFactors(gbm_sce, clusters = qclust)
+bladder_sce <- scran::computeSumFactors(bladder_sce, clusters = qclust)
 
 # Step 3) Normalize using these pooled sum factors and log transform. 
-gbm_sce <- scater::logNormCounts(gbm_sce)
+bladder_sce <- scater::logNormCounts(bladder_sce)
 ```
 
-## Compare normalized data to count data
+### Compare normalized data to count data
 
 One way to determine whether our normalization yields biologically relevant results is to plot it and see if similarly labeled samples and cells end up together.
 Because plotting 1000's of genes together isn't practical, we will reduce the dimensions of our data using Principal Components Analysis (PCA). 
```

Comment:
```suggestion
Because plotting 1000's of genes together isn't practical, we will reduce the dimensions of our data using Principal Components Analysis (PCA). 

We will also make the same plot with our unnormalized data, to visualize the effect of normalization on our sample.
Before plotting the unnormalized data, we will log transform the raw counts to make their scaling more comparable to the normalized data. 
To do this we will use the `log1p()` function, which is specifically designed for the case where we want to add 1 to all of our values before taking the log, as we do here.
(We could do something like `log(counts + 1)`, but this is both more efficient and more accurate.)
```
Github is being annoying and not letting me comment this on the next section which is what I wanted, but I think we could add a little more to the explanation of what's happening here to specify the comparison between unnormalized to normalized data in the intro. To me, it was a little confusing when first going through so I can imagine for someone not familiar with this they could also be confused. I added a few lines here that could replace the next paragraph to help with clarity.